agent/lm_agent/server_interfaces/lsdyna.py

```diff
@@ -37,7 +37,7 @@ async def get_output_from_server(self):
             output = await run_command(cmd)
 
             # try the next server if the previous didn't return the expected data
-            if output is None:
+            if not output:
```

Comment:
I wonder if getting the output via `run_command` should be put in the base-class or a mixin class so we don't implement it in each interface?

Reply:
The `run_command` method is going to be used by the reservation interface as well, that's why I moved it to a general `utils` module. Each server interface is calling the method from the `utils` module, not reimplementing it.

Do you think it's still worth adding the method in the base class, considering the reservation interface will need it too? If so, it would be outside the scope of this task.

Reply:
No. I didn't realize it was being used for other cases as well. Disregard!